xla/service/gpu/cublas_lt_matmul_thunk.cc

```diff
@@ -15,12 +15,17 @@ limitations under the License.
 
 #include "xla/service/gpu/cublas_lt_matmul_thunk.h"
 
+#if GOOGLE_CUDA || TF_HIPBLASLT
```

Comment:
Could this be handled in the build file instead?
So not use if_gpu_is_configured but if_cuda_is_configured and if_rocm_hipblaslt, same as for ir_emitter_triton.

Reply:
@akuegel I am sorry I am not sure I am 100% understand this one "be handled in the build file" do you mean to remove this preprocessor #if GOOGLE_CUDA || TF_HIPBLASLT in cubas_lt_matmul_thunk.cc ?  

the BUILD file for ```cublas_lt_matmul_thunk.cc``` and ```ir_emitter_triton.cc``` are using ```if_cuda_is_configured``` and ```if_rocm_hipblaslt```. 

https://github.com/openxla/xla/pull/3953/files#diff-35357ad3b3b70062889668b03399a71db3249cd2a01e90e0b0ed8ebe9c817272R1257-R1263

https://github.com/openxla/xla/pull/3953/files#diff-35357ad3b3b70062889668b03399a71db3249cd2a01e90e0b0ed8ebe9c817272R416-R417

Reply:
What I meant is: for triton_emitter target, your PR changes it to:

```
srcs = if_cuda_is_configured(["ir_emitter_triton.cc"])
        + if_rocm_hipblaslt(["ir_emitter_triton.cc"]),
```

I thought that the same might be possible for the cublas_lt_matmul_thunk target. Then, I think the preprocessor #if may not be needed.

Reply:
Got it! Grateful for your patience in explaining!